pypeit/core/skysub.py

```diff
@@ -470,6 +470,10 @@ def local_skysub_extract(sciimg, sciivar, tilts, waveimg, global_sky, rn2_img, t
      sobjs:
 
      Optional Args:
+     spat_pix: float ndarray, shape (nspec, nspat)
```

Comment:
I don't think 'Optional Args' is a valid `Google` doc string format.

And for sure it needs to be indented.

Yes, I am now getting anal on doc strings.
Partly because I just spent many many hours 
cleaning these up.